course/meetings/uoftcoders-2017-03-15.md

```diff
@@ -0,0 +1,98 @@
+---
+attendees: Madeleine, Luke, Lina, Joel, Lindsay, Elliot
+date: 2017-03-03
+---
+
+# Action items
+
+## Everyone (Finish before meeting next week)
+- _Most urgently_, two people edit and approve the blurb that I just pull requested before I send it off to CS https://github.com/UofTCoders/council/pull/101.
+- Read through the RQM course material and consider which strategy you prefer for introducing R-coding earlier:
+    1. Squish the first week into half the time.
+    2. Use the tutorial time to start the R-coding sessions from week 2 in parallelled with the week 1 lectures.
+- Find at least one data set that you think could be interesting for the students to work on.
+    - Share links to the data and check of your progress in this issue.
+    - Could add more later if we come across something, but it is good to have a solid foundation early.
+
+## Lina
+- Create a private Gitter room for the council under the current UofTCoders Gitter account.
+- Merge your minutes with my pull request here https://github.com/UofTCoders/council/pull/102.
+
+## Elliot
+- Give Joel write access where appropriate.
+    - Let me know what you need from me for this.
+
+## Joel
+- Email Asher course outline
+- Email Christie about managing groups and data sets
+- Email CS blurb
+
+# Course Outline
+We discussed the points Asher raised, and conceived of this initial outline of the course.
+
+## 1. Scheduling
+- One lecture and one tutorial of two hours each per week.
+- Preferably not on two consecutive days, so that students have time to study material in between sessions.
```

Comment:
@joelostblom was this changed eventually in your email you sent? I'll merge it in after it's added.